deployment/crawler-deployment.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  creationTimestamp: null
```

Comment:
please remove this key from all resources

Reply:
I will